database/sql/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -3,7 +3,7 @@ package cromwell.database.slick
 import java.sql.{Connection, PreparedStatement, Statement}
 import java.util.concurrent.{ExecutorService, Executors}
 
-import com.mysql.jdbc.exceptions.jdbc4.MySQLTransactionRollbackException
+import com.mysql.cj.jdbc.exceptions.MySQLTransactionRollbackException
```

Comment:
Who's CJ?

Reply:
"MySQL Connector/J" (wonder if it works on OS/2)